src/main/java/org/sefglobal/scholarx/service/MailInstance.java

```diff
@@ -0,0 +1,29 @@
+package org.sefglobal.scholarx.service;
+
+import org.simplejavamail.api.mailer.Mailer;
+import org.simplejavamail.mailer.MailerBuilder;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MailInstance {
```

Comment:
Shall we try to make this more meaningful?

Reply:
You mean about the class name right? @jayasanka-sack

Reply:
yes

Reply:
Fixed with [b7fe478](https://github.com/sef-global/scholarx/pull/345/commits/b7fe4789eab6ff7c620f2bd33ef4168506e110ff)